### PR TITLE
Update some GQL messages

### PR DESCRIPTION
**Pull request by WilcoNeo**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-217.surge.sh

Updated pages:
- [List of all GQLSTATUS server error codes](https://neo4j-docs-status-codes-217.surge.sh/status-codes/5/errors/gql-errors)

File: modules/ROOT/pages/errors/gql-errors.adoc [hunk 1]
```diff
@@ -535,6 +547,10 @@ Status description:: error: data exception - parsing JSON failure. Underlying er
 
 // Status description:: error: data exception - invalid property based access control rule involving WHERE and IS NULL. The property value access rule pattern `{ $pred }` always evaluates to 'NULL'. Use `WHERE' syntax in combination with `IS NULL` instead.
 
+=== 22NB1
+
+Status description:: error: data exception - type mismatch. Type mismatch: expected to be `{ $valueTypeOredList }` but was `{ $input }`.
```
Comment (Lojjs) [inline modules/ROOT/pages/errors/gql-errors.adoc:552]:
```suggestion
Status description:: error: data exception - type mismatch. Type mismatch: expected to be `{ $valueTypeList }` but was `{ $input }`.
```

We opted for not showing order in the docs but just document it with same list variables as in the code

**Review by Lojjs — COMMENTED** [review]
Just one minor thing, otherwise it looks good

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-217.surge.sh

Updated pages:
- [List of all GQLSTATUS server error codes](https://neo4j-docs-status-codes-217.surge.sh/status-codes/5/errors/gql-errors)

**Review by Lojjs — APPROVED** [review]
🙇

**Pull request merged**